examples/redux/README.md

```diff
@@ -0,0 +1,41 @@
+# Usage with Redux
+
+#### 1. Add dependencies
+``` bash
+npm install airbrake-js redux-airbrake --save
+```
+
+#### 2. Import dependency
+``` js
+import airbrakeMiddleware from 'redux-airbrake';
+```
+
+#### 3. Configure & add middleware
+``` js
+const errorMiddleware = airbrakeMiddleware({
+    projectId: '******',
+    projectKey: '**************'
+});
```

Comment:
The middleware is actually using airbrake-js client behind the scenes. See its implementation here: https://github.com/alexcastillo/redux-airbrake/blob/master/src/index.js#L12

It already supports configuration for filters, etc.